harness/determined/exec/harness.py

```diff
@@ -136,14 +136,15 @@ def build_and_run_training_pipeline(env: det.EnvContext) -> None:
                 if env.experiment_config.debug_enabled():
                     faulthandler.dump_traceback_later(30, repeat=True)
 
-                controller = load.prepare_controller(
-                    env,
-                    iter(workload_mgr),
-                    load_path,
-                    socket_mgr.get_rendezvous_info(),
-                    hvd_config,
-                )
-                controller.run()
+                with det._execution_manager():
```

Comment:
I don't think this is a good idea.  This harness.py entrypoint does not need sys.path editing.  That is purely a way to undo the sys.path editing that our CLI entrypoint did.

Can we just make this `_catch_sys_exit()` instead, since that's actually what needs to happen here?

Reply:
done.